base/shell/explorer/trayclock.cpp

```diff
@@ -267,6 +267,10 @@ WORD CTrayClockWnd::GetMinimumSize(IN BOOL Horizontal, IN OUT PSIZE pSize)
     if (!LinesMeasured)
         return 0;
 
+    /* Prevents the date from being cut off when the day of the week is shorter than the date. */
```

Comment:
```suggestion
    /* Prevents the date from being cut off when the day of the week is shorter than the date */
```
By "date" you mean the "dd/mm/yyyy" thing (first line of the clock in the screenshots)?